Makefile

```diff
@@ -184,6 +222,26 @@ test-timers:
 test-timers-clean:
 	$(MAKE) --directory=tools clean
 
+test-v8:
+	# note: performs full test unless QUICKCHECK is specified
+	deps/v8/tools/run-tests.py --arch=$(ARCH) --mode=$(BUILDTYPE_LOWER) $(V8_TEST_NO_I18N) \
+	  $(QUICKCHECK_ARG) --no-presubmit --shell-dir=deps/v8/out/$(ARCH).$(BUILDTYPE_LOWER) \
+	 $(TAP_V8)
+
+test-v8-intl:
+	# note: performs full test unless QUICKCHECK is specified
+	deps/v8/tools/run-tests.py --arch=$(ARCH) --mode=$(BUILDTYPE_LOWER) --no-presubmit \
+	  $(QUICKCHECK_ARG) --shell-dir=deps/v8/out/$(ARCH).$(BUILDTYPE_LOWER) intl $(TAP_V8_INTL)
+
+test-v8-benchmarks:
+	# note: this runs with --download-data so it'll go out and
+	deps/v8/tools/run-tests.py --arch=$(ARCH) --mode=$(BUILDTYPE_LOWER) --download-data \
+	  $(QUICKCHECK_ARG) --no-presubmit --shell-dir=deps/v8/out/$(ARCH).$(BUILDTYPE_LOWER) benchmarks \
+	 $(TAP_V8_BENCHMARKS)
+ 
+test-v8-all: test-v8 test-v8-intl test-v8-benchmarks
```

Comment:
Added v8 make command as well as all test commands (test-v8, test-v8-intl, test-v8-benchmarks, test-v8-all) to .PHONY.